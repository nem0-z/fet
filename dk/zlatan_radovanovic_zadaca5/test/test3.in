print 3.25 + 10.5 * { 5.60 - 12.60 }
print 3.50 + 10.100 * { 5 - 12 / (4 - 2) } + [100 + 100]
