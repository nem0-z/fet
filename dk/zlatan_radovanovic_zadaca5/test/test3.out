-70.250000
193.400000
